src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -107,22 +107,40 @@ extern "C" {
  * (0x56000000)
  *  @{
  */
-#define STATUS_SDP_BASE                         STATUS_WEBRTC_BASE + 0x01000000
-#define STATUS_SDP_MISSING_ITEMS                STATUS_SDP_BASE + 0x00000001
-#define STATUS_SDP_ATTRIBUTES_ERROR             STATUS_SDP_BASE + 0x00000002
-#define STATUS_SDP_BANDWIDTH_ERROR              STATUS_SDP_BASE + 0x00000003
-#define STATUS_SDP_CONNECTION_INFORMATION_ERROR STATUS_SDP_BASE + 0x00000004
-#define STATUS_SDP_EMAIL_ADDRESS_ERROR          STATUS_SDP_BASE + 0x00000005
-#define STATUS_SDP_ENCYRPTION_KEY_ERROR         STATUS_SDP_BASE + 0x00000006
-#define STATUS_SDP_INFORMATION_ERROR            STATUS_SDP_BASE + 0x00000007
-#define STATUS_SDP_MEDIA_NAME_ERROR             STATUS_SDP_BASE + 0x00000008
-#define STATUS_SDP_ORIGIN_ERROR                 STATUS_SDP_BASE + 0x00000009
-#define STATUS_SDP_PHONE_NUMBER_ERROR           STATUS_SDP_BASE + 0x0000000A
-#define STATUS_SDP_TIME_DECRYPTION_ERROR        STATUS_SDP_BASE + 0x0000000B
-#define STATUS_SDP_TIMEZONE_ERROR               STATUS_SDP_BASE + 0x0000000C
-#define STATUS_SDP_URI_ERROR                    STATUS_SDP_BASE + 0x0000000D
-#define STATUS_SDP_VERSION_ERROR                STATUS_SDP_BASE + 0x0000000E
-#define STATUS_SDP_ATTRIBUTE_MAX_EXCEEDED       STATUS_SDP_BASE + 0x0000000F
+#define STATUS_SDP_BASE                                   STATUS_WEBRTC_BASE + 0x01000000
+#define STATUS_SDP_MISSING_ITEMS                          STATUS_SDP_BASE + 0x00000001
+#define STATUS_SDP_ATTRIBUTES_ERROR                       STATUS_SDP_BASE + 0x00000002
+#define STATUS_SDP_BANDWIDTH_ERROR                        STATUS_SDP_BASE + 0x00000003
+#define STATUS_SDP_CONNECTION_INFORMATION_ERROR           STATUS_SDP_BASE + 0x00000004
+#define STATUS_SDP_EMAIL_ADDRESS_ERROR                    STATUS_SDP_BASE + 0x00000005
+#define STATUS_SDP_ENCYRPTION_KEY_ERROR                   STATUS_SDP_BASE + 0x00000006
+#define STATUS_SDP_INFORMATION_ERROR                      STATUS_SDP_BASE + 0x00000007
+#define STATUS_SDP_MEDIA_NAME_ERROR                       STATUS_SDP_BASE + 0x00000008
+#define STATUS_SDP_ORIGIN_ERROR                           STATUS_SDP_BASE + 0x00000009
+#define STATUS_SDP_PHONE_NUMBER_ERROR                     STATUS_SDP_BASE + 0x0000000A
+#define STATUS_SDP_TIME_DECRYPTION_ERROR                  STATUS_SDP_BASE + 0x0000000B
+#define STATUS_SDP_TIMEZONE_ERROR                         STATUS_SDP_BASE + 0x0000000C
+#define STATUS_SDP_URI_ERROR                              STATUS_SDP_BASE + 0x0000000D
+#define STATUS_SDP_VERSION_ERROR                          STATUS_SDP_BASE + 0x0000000E
+#define STATUS_SDP_ATTRIBUTE_MAX_EXCEEDED                 STATUS_SDP_BASE + 0x0000000F
+#define STATUS_SDP_UNKNOWN_ERROR                          STATUS_SDP_BASE + 0x00000010
+#define STATUS_SDP_BAD_PARAM                              STATUS_SDP_BASE + 0x00000011
```

Comment:
Instead of adding this, shouldn't it be mapped to STATUS_INVALID_ARG

Reply:
Thanks for suggestion. Updated for this.